app/main.py

```diff
@@ -25,19 +25,21 @@
 from slowapi import _rate_limit_exceeded_handler
 from slowapi.errors import RateLimitExceeded
 
+from .auth import authlib_oauth
 from .controllers import auth_router, example_router, health_router
 from .settings import api_settings, rate_limiter
 
 fastAPI_logger = logger  # convenient name
 
 
 def get_application() -> FastAPI:
-    tags_metadata = [
-        {
-            "name": "apikeymanager",
-            "description": "Operations with users. The **login** logic is also here.",
-        },
-    ]
+    # tags_metadata = [
```

Comment:
Do we really need this section ? It adds a text description to the swagger, I'm not sure it's useful, but it's purely cosmetic.